src/ray/protobuf/common.proto

```diff
@@ -552,3 +552,21 @@ message NamedActorInfo {
   string ray_namespace = 1;
   string name = 2;
 }
+
+// A batch of multiple log lines.
+message LogBatch {
```

Comment:
Feel free to suggest alternative files for this.

Reply:
Will this be used in something other than PubSub? If not, putting it in `pubsub.proto` may be better.